docs/modules/ROOT/pages/api/finance.adoc

```diff
@@ -91,7 +91,7 @@ Returns the total vested amount of a specified `token` at a given `timestamp`.
 
 Releases the amount of a given `token` that has already vested and returns that amount.
 
-Emits {AmountReleased} event.
+Emits an {AmountReleased} event.
```

Comment:
Should we amend this to "may emit" as well? It appears to be an unfaithful implementation if the interface says "emits" while the impl of the interface says "may emit"